pkg/dependencies/factory.go

```diff
@@ -253,7 +253,7 @@ func (f *Factory) UseExecutablesDockerClient(client executables.DockerClient) *F
 
 // dockerLogin performs a docker login with the ENV VARS.
 func dockerLogin(ctx context.Context, registry string, docker executables.DockerClient) error {
-	username, password, _ := config.ReadCredentials()
+	username, password, _ := registrymirror.Credentials()
```

Comment:
Any particular reason the error isn't being checked?

Reply:
If either of them is not set, the error gets caught immediately by the next line `err := docker.Login(ctx, registry, username, password)`.

Reply:
Yea I don't see a reason not to check the error; it gives an opportunity to create a much clearer error.